data/config.default.toml

```diff
@@ -0,0 +1,187 @@
+# Smoked Salmon Configuration File
+# Read details about each setting here:
+# https://github.com/smokin-salmon/smoked-salmon/wiki/Configuration
+
+# ===============================
+# DIRECTORY SETTINGS
+# ===============================
+[directory]
+# Where to copy files (usually the download/seeding folder for your torrent client)
+download_directory = '/path/to/folder/for/the/releases'
+
+# Hardlinks are enabled by default, set the option below to true to DISABLE hardlinks (and use standard copy files)
+# hardlinks = false
```

Comment:
This is either a bad name, or a bad option.

You could either:
* rename it to `disable_hardlinks`
* make it default to `true`, and disabled on `false` (my preferred option)

Reply:
Ah okay, then the comment could explicitly say "set to false".